### DISCUSSION TITLE
Any change to "convert" rpm info to celcius?

### DISCUSSION THREAD

**Discussion by lassivv** [Rem0o/FanControl.Releases#470] [Q&A]
Weird question i know :) 

I need to get my waterloop water temperature to my HWinfo and off course FanControl and give control radiator fans with that info.

Problem is i dont have waterloop sensor place on my motherboard. Thats why i do simple conventer via Arduino. I plug water temparature sensor (NTC) to my arduino and do little mosfet circuit to give "tachometer" reading out arduino output pin and i plugged that my motherboard.

Now i get numbers my hwinfo and fancontrol 

Chassis fan 4:
2000rpm = 20C water temp
3000rpm = 30C water temp
4000rpm = 40C water temp
etc.

Just interested can i some way convert that rpm info to celcius info example that custom sensor option or someway else? This is very easy and cheap way to get my water temp showing, but now only thing is get that control my fans via FanControl :)

**Comment by redbaron** [comment]
I am facinng same problem and it seems there are 2 options, both require writing a FanControl plugin:
- Create a plugin reading tach with libhardwaremonitor same as FanControl itself is doing it , but producing temperature reading as a "new" sensor.
- Create  plugin to read sensor data from COM (serial) interface where your arduino prints it.

If you go with tach route, how do you power arduino? Can it take 12V from pin header?
